_posts/2015-12-29-adding-docker-to-a-ruby-gem.markdown

```diff
@@ -0,0 +1,54 @@
+---
+layout: post
+title: "Adding Docker to a Ruby gem"
+date: 2015-12-29 13:55:00
+categories: ["docker"]
```

Comment:
I think the post warrants an `open-source` tag, and _maybe_ a `ruby` tag as well, what do you think?